packages/field-plugin/helpers/vite/src/sandbox.ts

```diff
@@ -58,4 +104,4 @@ const displayManifestChecking = () => {
 }
 
 const displayManifestErrorLoading = (err: Error) =>
-  console.log(`${arrows.red} ${bold(`${err.message}`)}`)
+  console.log(`${arrows.red} ${red(`${err.message}`)}`)
```

Comment:
I added this for better error visibility as when the text is black it is hard to spot